### PR TITLE
no space after password

### PR DESCRIPTION
**Pull request by crami**
The space after Password is not present in the prompt on all devices I checked.

### CODE REVIEW

**Comment by ytti** [comment]
Either talk to person who wrote that line, or err towards safety and guard the space with ```?```

**Comment by ytti** [comment]
bump

**Comment by ytti** [comment]
bump

File: README.md [hunk 1]
```diff
@@ -508,7 +508,7 @@ source:
     map:
       name: ip
       model: model
-      username: username
+      user: username
```
Comment (danilopopeye) [inline README.md]: @crami the `name` change should be the one at line `506`. could you fix this, please?

ref.: [lib/oxidized/source/sql.rb](https://github.com/ytti/oxidized/blob/c8ca088c1314bc55284fd6bc110a2372d22b1ce4/lib/oxidized/source/sql.rb#L53-L57)

**Comment by laf** [comment]
@danilopopeye I've fixed the user/username issue in the docs.

LGTM now.

**Pull request merged**